src/pybind/mgr/cephadm/module.py

```diff
@@ -2770,6 +2779,47 @@ def _create_rgw(self, rgw_id, host):
     def apply_rgw(self, spec):
         return self._apply(spec)
 
+    @trivial_completion
```

Comment:
```suggestion
```

Looks like this was reverted by https://github.com/ceph/ceph/pull/34260

Reply:
Cool, will remove it. FYI according the #34268 add_nfs still seems to have it.

Reply:
good catch! Fixed by #34361